doc/rst/technotes/libraries.rst

```diff
@@ -111,23 +205,145 @@ example looks like this:
    myCProg: myCProg.c lib/libfoo.a
      $(CHPL_COMPILER) $(CHPL_CFLAGS) -o myCProg myCProg.c $(CHPL_LDFLAGS)
 
+Makefile-less Compilation
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Alternatively, you can get the right combination is by using the ``compileline
+Alternatively, you can get the right combination by using the ``compileline
 --compile`` and ``compileline --libraries`` tools we provide.  The compilation
-command would then look like this (replacing myprog.c with the name of your C
-program):
+command would then look like this (replacing myCProg.c with the name of your C
+program that will use the library):
 
 .. code-block:: sh
 
-   `$CHPL_HOME/util/config/compileline --compile` myprog.c -L. -lfoo `$CHPL_HOME/util/config/compileline --libraries`
+   `$CHPL_HOME/util/config/compileline --compile` myCProg.c -Llib/ -lfoo `$CHPL_HOME/util/config/compileline --libraries`
 
 Note that ``compileline --compile-c++`` is also available. It requests a
 compilation command able to compile a C++ program.
 
+Using Your Library in Python
+============================
+
+Pre-requisites
+--------------
+
+To make use of your library in Python with minimal work, the Chapel compiler
+requires that you have ``python3`` installed in your ``$PATH``, and that your
+``python3`` installation contains ``numpy`` and ``Cython``.
+
+We don't anticipate supporting ``python2`` at this time, sorry.
```

Comment:
(Un)planned features would be better documented in issues rather than here. 

Side note: It bothers me that we only support python2 some places and only python3 here. I think the solution is to support python3 everywhere, but we're in an awkward position right now.

Reply:
Yeah, it would be nice to go on a clean up spree and update everything to use python3.  Maybe this is a conversation for the group meeting about what we need to still support?

Reply:
Probably not much has changed, though :/